projects/js-packages/ai-client/CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.33.20] - 2025-11-17
+### Changed
+- Update package dependencies. [#45915]
+- Update package dependencies. [#45958]
```

Comment:
We usually combine these into one like this

Update package dependencies. [#45915] [#45958]